assets/sass/widgets/_googlesitekit-widget-searchFunnel.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Search Funnel styles.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-cta--analytics {
+		--cta-analytics-gap: #{$grid-gap-phone};
+
+		@media (min-width: $bp-tablet) {
+			--cta-analytics-gap: #{$grid-gap-desktop};
+		}
+
+		background-color: $c-analytics-cta-bg;
+		display: flex;
+		flex-wrap: wrap-reverse;
+		font-size: 14px;
+		gap: var(--cta-analytics-gap);
+		padding: var(--cta-analytics-gap);
+
+		.mdc-button {
+			display: block;
+			letter-spacing: normal;
```

Comment:
@techanvil Let's use the default letter spacing for the button to be consistent with the existing styles. The same applies for `text-transform`, let's use what we already have currently.

Reply:
Thanks for this suggestion. I'm happy with the idea of reusing the existing styles for design consistency, but want to bring your attention to the Figma design where it's got lowercase text and the spacing is "normal" too. Have you taken this into consideration with regard to your proposal?

![image](https://user-images.githubusercontent.com/18395600/155170924-89ae7433-d0d7-447b-abb6-0be0970c8994.png)

Reply:
Cool, thanks for letting me know. I've reverted these letter styling adjustments.